### ISSUE TITLE
simpler c++ code

### ISSUE THREAD

**Issue by rspadim** [pplonski/keras2cpp#42]
Hi guys, could we create a simpler c++ code? 

maybe a plain C code and avoid external libs and file read

my idea is a fixed code with hardcoded parameters values, and only predicting "one row", no batch prediction and no GPU support, something very very simple

if possible don't use vector<> just use fixed arrays, something like treelite do with xgboost

only one function to predict + inline functions to activation functions sigmoid, relu, conv, etc:

double predict(double *data){
}

data is an array of input "features"

**Comment by pplonski** [comment]
Nice idea, go ahead!
